src/main/typescript/node_modules/@atomist/rug/test/project/WellKnownSteps.ts

```diff
@@ -88,6 +97,11 @@ Then("file at ([^ ]+) should contain (.*)", (p, w, path: string, lookFor: string
 });
 
 /**
- * When step should fail
+ * When step should fail.
  */
 Then("it should fail", (p, world) => world.failed());
+
+/**
+ * The scenario was aborted due to an exception being thrown.
+ */
+Then("the scenario aborted", (p, w) => w.aborted());
```

Comment:
I actually started to put them in but stopped after realizing you did not when you touched this file for #566/#658. I figured you might have had a reason for not adding them then. If not, I'm happy to add them.